framework/wazuh/core/utils.py

```diff
@@ -1363,7 +1363,8 @@ class WazuhDBQuery(object):
     def __init__(self, offset: int, limit: int, table: str, sort: dict, search: dict, select: list, query: str,
```

Comment:
This is what I was meaning when said class attribute (`instead of instance attribute`)
```suggestion
    JOIN_TYPES = ['LEFT', 'CROSS', 'OUTER']

    def __init__(self, offset: int, limit: int, table: str, sort: dict, search: dict, select: list, query: str,
```

Reply:
My bad, fixed in https://github.com/wazuh/wazuh/pull/17950/commits/69228c4d5ec07e8864be8d2f7d1253ad760107e5